homework/style.css

```diff
@@ -1,3 +1,105 @@
-.alert-error {
```

Comment:
Your design is not responsive:

![image](https://user-images.githubusercontent.com/2788771/61939444-8fda1c00-af93-11e9-8fc1-b41f21c3159e.png)